tests/resources/rules/rules.json

```diff
@@ -77113,176 +77113,80 @@
                         "results_present_sql": true,
                         "results_sql": [
                             {
-                                "dataset": "ae",
+                                "dataset": "ae.xpt",
                                 "domain": "AE",
-                                "execution_status": "success",
-                                "execution_message": "Negative value for AEDUR",
+                                "execution_status": "execution_error",
+                                "execution_message": "rule execution error",
                                 "number_errors": 1,
                                 "errors": [
                                     {
-                                        "row": 1,
-                                        "SEQ": 1,
-                                        "USUBJID": "CDISC001",
-                                        "value": {
-                                            "AEDUR": "P-1DT2H"
-                                        }
+                                        "error": "An unknown exception has occurred",
```

Comment:
These errors look bad, but it's not clear how you could have caused them with this PR?